src/main/java/org/openstreetmap/atlas/geography/atlas/items/Area.java

```diff
@@ -77,6 +79,19 @@ public boolean intersects(final GeometricSurface surface)
         return surface.overlaps(asPolygon());
     }
 
+    @Override
+    public boolean isCountrySliced()
+    {
+        // Get the last 6 digits of the identifier
+        final String stringIdentifier = String.valueOf(this.getIdentifier());
+        final int lastSix = stringIdentifier.length() > IDENTIFIER_SUFFIX_LENGTH ? Integer.parseInt(
```

Comment:
Could we expect any Area to have an identifier with less than 6 length that's still country sliced?

Reply:
No, part of country slicing is to append the 6 digit suffix to all objects.

Reply:
Got it